lldb/source/Plugins/Process/Linux/NativeProcessLinux.cpp

```diff
@@ -1641,6 +1641,10 @@ NativeProcessLinux::GetSoftwareBreakpointTrapOpcode(size_t size_hint) {
 
 Status NativeProcessLinux::ReadMemory(lldb::addr_t addr, void *buf, size_t size,
                                       size_t &bytes_read) {
+  Log *log = GetLog(POSIXLog::Memory);
+  LLDB_LOG(log, "addr = {0}, buf = {1}, size = {2}", addr, buf, size);
```

Comment:
Does this include the function name? If not, add something to say it's a read.

Reply:
it does (with `log enable -F`)